src/cirq_iqm/iqm_device.py

```diff
@@ -310,6 +310,7 @@ def optimization_at(
             index: int,
             op: cirq.Operation,
     ) -> Optional[cirq.PointOptimizationSummary]:
+        """TODO the parent class at Cirq has a broken docstring here, which we have to override."""
```

Comment:
Can we add a link to the corresponding Cirq issue for fixing the broken docstring?

Reply:
I haven't reported it. There are some broken docstrings in Cirq, but they only cause us trouble because of the "treat warnings as errors" flag we use when building docs. I think Cirq currently uses a custom API docs generation system from TensorFlow, so it may not be a problem for them.

Reply:
Another solution would be to not inherit any docstrings from Cirq parent classes, but IMO that would make the docs less useful to the user.

Reply:
Ok. If we write here that some Cirq docstrings are broken I think we should report it as well. Or if it isn't worth reporting then maybe we don't write that they're broken?

Reply:
I reported it, https://github.com/quantumlib/Cirq/issues/4276

Reply:
```suggestion
        """TODO the parent class at Cirq has a broken docstring here, which we have to override."""
        """https://github.com/quantumlib/Cirq/issues/4276"""
```